x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/create_rules_route.ts

```diff
@@ -120,7 +123,17 @@ export const createCreateRulesRoute = (server: ServerFacade): Hapi.ServerRoute =
           references,
           version: 1,
         });
-        return transformOrError(createdRule);
+        const ruleStatuses = await savedObjectsClient.find<
+          IRuleSavedAttributesSavedObjectAttributes
+        >({
+          type: ruleStatusSavedObjectType,
+          perPage: 10,
```

Comment:
This should work with `perPage: 1` I would change it now and ensure that works out. Getting 10 per page to toss away 9 seems wasteful.